### PR TITLE
Correct spelling of Red Canary and add prior year's reports to the list

### PR DESCRIPTION
**Pull request by keithmccammon**

### CODE REVIEW

**Review by jacobdjwilson — CHANGES_REQUESTED** [review]
I've uploaded 2020 and 2021 reports to the github sub-directory in this repo. README.md only displays the latest report.
Thanks for the spelling catch! If you want to resubmit with only the latest report link with correct spelling I'm happy to accept the commit.

**Review by paks000 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -47,7 +47,10 @@ Reports have been classified into two categories by the source of data.
 |[Proofpoint](https://www.proofpoint.com/us/resources/threat-reports/state-of-phish)|[State of the Phish](Annual%20Security%20Reports/2022/Proofpoint-State-of-the-Phish-2022.pdf)|2022|Attacks|
 |[Proofpoint](https://www.proofpoint.com/us/resources/threat-reports/cost-of-insider-threats)|[Cost of Insider Threats Global Report](Annual%20Security%20Reports/2022/Proofpoint-Ponemon-Cost-of-Insider-Threats-Global-Report-2022.pdf)|2022|Industry|
 |[Rapid7](https://www.rapid7.com/research/report/2021-industry-cyber-exposure-report/)|[Industry Cyber Exposure Report](Annual%20Security%20Reports/2021/Rapid7-Industry-Cyber-Exposure-Report-2021.pdf)|2021|Attacks|
-|[RedCanary](https://redcanary.com/threat-detection-report/)|[Threat Detection Report](Annual%20Security%20Reports/2022/RedCanary-Threat-Detection-Report-2022.pdf)|2022|Attacks|
```
Comment (paks000) [inline README.md:50]: RED CANARY

**Review by paks000 — CHANGES_REQUESTED** [review]
correct spelling of red carnary

File: README.md (hunk 1 above)
Comment (jacobdjwilson) [reply README.md:50]: 2019, 2020, and 2021 reports were added to the repository 3 months ago by @keithmccammon 
Spelling has also ben resolved

**Comment by jacobdjwilson** [comment]
Legacy reports added to repository and spelling mistakes fixed.

**Pull request closed without merging**